modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf

```diff
@@ -22,17 +22,21 @@
 
 		iframe.on('load', () => {
 			var height = A.Plugin.AutosizeIframe.getContentHeight(iframe);
+			var heightNormal = 600;
```

Comment:
Hey @locpham97, what about renaming this variable to defaultHeight?

Reply:
hello @diegonvs , it is referred to as the existing variable in this file https://github.com/liferay-core-infra/liferay-portal/blob/master/modules/apps/iframe/iframe-web/src/main/java/com/liferay/iframe/web/internal/configuration/IFramePortletInstanceConfiguration.java line 84.

Reply:
Oh ok. I got it.